app/services/partnerships/update.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Partnerships
+  class Update
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+
+    attribute :partnership
+    attribute :delivery_partner_id
+
+    validates :partnership, :delivery_partner_id, presence: true
+    validate :validate_delivery_partner
+
+    def call
+      return false if invalid?
+      return true if partnership.delivery_partner == delivery_partner
```

Comment:
I think we'll need to return the partnership here otherwise it breaks, could you add a test as well?

Reply:
Added test

Reply:
Added test